cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/KafkaConnectService.java

```diff
@@ -276,4 +309,52 @@ protected ClusterStatus getKafkaConnectStatus(
       return ClusterStatus.OFFLINE;
     }
   }
+
+  public ApiResponse restartConnector(ClusterConnectorRequest clusterConnectorRequest) {
+    log.info("Into restartConnector clusterConnectorRequest {} ", clusterConnectorRequest);
+
```

Comment:
Can we put in a permission check here? I think this should probably be an elevated non default permission?

Reply:
this is cluster api. I have already added a permission for managing connectors